src/renderer/react/app/login/Login.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { injectIntl, intlShape } from 'react-intl';
+import { notify, STATUS } from 'reapop';
+
+import { loginAction } from 'redux/app/actions';
+
+import LoginPresentation from './LoginPresentation';
+import messages from './login.messages';
+
+const mapDispatchToProps = dispatch => ({
+  actions: bindActionCreators({
+    login: loginAction,
+    notify,
+  }, dispatch),
+});
+
+export const LoginContainer = (Presentation) => {
```

Comment:
What's the point of writing an HOC here ?

Reply:
Is it not a best practice ?

Reply:
I think it's a best practice when the code from the HOC will be re-used, or else it just adds useless complexity (but it's my POV)

Reply:
Well the thing is, I believe it will be easier to unit test both parts this way :D

Reply:
@sylvainar It's easier to think of your container as something completely isolated from the presentation when you write them as HOC with the presentation as parameter.